app.py

```diff
@@ -1,72 +1,186 @@
-#!/bin/env python
+#!/usr/bin/env python
 """
 This example uses docopt with the built in cmd module to demonstrate an
 interactive command application.
+
 Usage:
-    dojo create_room <room_type> <room_names> ...
-    dojo add_room (<fname> <lname> <FELLOW/STAFF> [<wants_accommodation>])
-    dojo [-i | --interactive]
-    dojo
-    dojo (-h | --help | --version)
+    app.py
+    app.py create_room <room_type> <room_names>...
+    app.py add_person <fname> <lname> <FELLOW/STAFF> [<wants_accommodation>]
+    app.py print_room <room_name>
+    app.py print_allocations [(-o <filename>)]
+    app.py print_unallocated [(-o <filename>)]
+    app.py people_id
+    app.py reallocate_person <person_identifier> <room_name>
+    app.py load_people <filename>
+    app.py save_state [(--db sqlite_database)]
+    app.py load_state <sqlite_database>
+    app.py (-i | --interactive)
+    app.py (-h | --help | --version)
+
 Options:
     -i, --interactive  Interactive Mode
     -h, --help  Show this screen and exit.
 """
 
-#import necessary modules
 import sys
 import cmd
```

Comment:
You could improve on your import order here. Check this [ref](https://www.python.org/dev/peps/pep-0008/#imports)